tonic/src/transport/server/incoming.rs

```diff
@@ -27,7 +28,19 @@ where
         let mut incoming = pin!(incoming);
 
         while let Some(item) = incoming.next().await {
-            yield item.map(ServerIo::new_io)?
+            yield match item {
+                Ok(_) => item.map(ServerIo::new_io)?,
+                Err(e) => {
+                    let e = e.into();
+                    tracing::debug!(error = %e, "accept loop error");
+                    if let Some(e) = e.downcast_ref::<io::Error>() {
+                        if e.kind() == io::ErrorKind::ConnectionAborted {
+                            continue;
+                        }
+                    }
+                    Err(e)?
+                }
+            }
         }
     }
```

Comment:
```
Ok(_) => item.map(ServerIo::new_io)?
```

could also be written as

```
Ok(io) => Ok::<ServerIo<IO>, crate::Error>(ServerIo::new_io(io))?,
```

but in terms of readability and diff, I opted for the former (any better ideas?).